app/assets/javascripts/provider/admin/cms/intro.js.coffee

```diff
@@ -21,8 +21,8 @@ create_intro_tabs = () ->
   try
     jQueryUI('#cms-intro-tabs').tabs
       active: $("#cms-intro-tabs > ##{tab}").data('ui-tabs-index')
-      activate: (event,ui) -> 
-        currentTab = ui.newPanel.selector.slice(1)
+      activate: (event,ui) ->
+        currentTab = ui.newPanel.selector.slice(1) # FIXME: don't use .selector on jquery objects (jquery 1.9)
```

Comment:
Probably good news is that this piece is not actually working :sweat_smile: 
This is from prod:
![image](https://github.com/3scale/porta/assets/1270328/e410d25d-f558-47e6-94b0-8deabde6ffe9)

Apparently nobody noticed :grimacing:  so maybe we can just drop it :upside_down_face:

Reply:
> What's the alternative to `.selector`? Would it require too much work to fix the `FIXME`?

There is no alternative (or there won't be, in 3.x) so the FIXME is not obvious and depend on each case. However, I don't think it would be much work.

> Probably good news is that this piece is not actually working 😅 This is from prod: [image]
> Apparently nobody noticed 😬 so maybe we can just drop it 🙃

Many broken pieces of javascript that are being dug up... I will make a note of this file in [THREESCALE-10943: Clean up javascript assets](https://issues.redhat.com/browse/THREESCALE-10943).